src/components/overlay/Overlay.js

```diff
@@ -90,7 +83,6 @@ class Overlay extends PureComponent {
               onKeyDown={this.handleEscKey}
               onMouseDown={this.handleMouseDown}
               onMouseUp={this.handleMouseUp}
-              onClick={this.handleClick}
```

Comment:
Missing of `props.onClick` is okay?

Reply:
Yes, if passed it will be set through `{...other}`.